### ISSUE TITLE
Send data to widgets

### ISSUE THREAD

**Issue by marionschleifer** [simplificator/monitoring-dashboard#9]
I have included the notifier class from kitto: https://github.com/kittoframework/kitto/blob/master/lib/kitto/notifier.ex

I tried to use it in the following file: monitoring_dashboard/lib/jobs/semaphore. There is no error message anymore, but the data is not displayed in the widget. 

Other question: Where should I start (start_link) the notifier class?

**Comment by crackofdusk** [comment]
@marionschleifer There is no need for the `Notifier` module. We have replaced it by the Phoenix PubSub library. To publish the data, use `MonitoringDashboard.Web.Endpoint.broadcast!`, like you already do in the polling GenServers.

**Comment by marionschleifer** [comment]
@crackofdusk Thank you. Where should I do that? In the `send_message` method in the `event_controller`? because there, I subscribe for this data and also receive it. I suppose the `broadcast!` will look exactly like I use it in the polling GenServers (`PubSub.broadcast!("new_relic", "status_check", %{items: list})`)?

**Comment by crackofdusk** [comment]
No: for polling, the data is already connected. If something is wrong, I would debug what the value of `message` is in the [`send_message` function](https://github.com/simplificator/monitoring-dashboard/blob/43222bbfaf7f83889b3bf1afab8cf0d5b8261dc3/apps/monitoring_dashboard_web/lib/monitoring_dashboard_web/controllers/event_controller.ex#L16)

If the data data is not valid json, you will have to use `Poision.encode` to transform the elixir structs to json.
